cli/Cargo.toml

```diff
@@ -37,7 +37,7 @@ regex = "1.2.1"
 remove_dir_all = "0.5.2"
 ring = "~0.14.6"
 rustyline = "5.0.2"
-serde = "1.0.99"
+serde = { version = "1.0.99", features = ["derive"] }
```

Comment:
Preferrably do this as a separate PR, so we can keep the cargo an gn builds in sync.

Reply:
#2808